### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by paulabernathy7**
MVP incomplete

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -63,32 +63,55 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );
```
Comment (Dilstom) [inline assignments/arrays.js:66]: yup, this is how we use template literals )

File: assignments/arrays.js [hunk 2]
```diff
@@ -63,32 +63,55 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[49];
+console.log(lastCar.car_make, lastCar.car_model);
```
Comment (Dilstom) [inline assignments/arrays.js:71]: or: 
`lastCar = inventory[inventory.length - 1];`

File: assignments/arrays.js [hunk 3]
```diff
@@ -63,32 +63,55 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+let myCarModels = [];
+for (let i = 0; i < inventory.length; i ++) {
+    console.log(inventory[i].car_model)
+    myCarModels.push(inventory[i].car_model);
+}
+console.log(myCarModels.sort());
```
Comment (Dilstom) [inline assignments/arrays.js:80]: very good! when you want to check what is happening in each step, just console log it )

File: assignments/arrays.js [hunk 4]
```diff
@@ -63,32 +63,55 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
+console.log(oldCars.length);
 
 // ==== Challenge 6 ====
 // A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
 let BMWAndAudi = [];
-console.log();
+
+for (let i = 0; i < inventory.length; i ++) {
+    if (inventory[i].car_make === "BMW" || "Audi"){
+        BMWAndAudi.push(inventory[i])
```
Comment (Dilstom) [inline assignments/arrays.js:107]: yeah, I don't think this is going to work the way you want. Run your code and check the result.

File: assignments/function-conversion.js [hunk 5]
```diff
@@ -1,21 +1,21 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
+// let myFunction =  () => {
 // console.log("Function was invoked!");
 // };
```
Comment (Dilstom) [inline assignments/function-conversion.js:5]: or just in one line: 
`let myFunction = _ => console.log('Function was invoked!');`  :)

File: assignments/function-conversion.js [hunk 6]
```diff
@@ -1,21 +1,21 @@
...
-// let anotherFunction = function (param) {
+// let anotherFunction = (param) => {
 //   return param;
 // };
 // anotherFunction("Example");
 
-// let add = function (param1, param2) {
+// let add = (param1, param2) => {
 //   return param1 + param2;
 // };
```
Comment (Dilstom) [inline assignments/function-conversion.js:15]: very good! or, again:
`let add = (param1, param2) => param1 + param2;`

File: assignments/objects.js [hunk 7]
```diff
@@ -19,27 +19,84 @@ const example = {
...
+  "gender": 'M'
+}
+
+const intern5 = {
+  "id": 5,
+  "name": 'Antonietta',
+  "email": 'adaine5@samsung.com',
+  "gender": 'F',
+  "multiplyNums": function (a,b) {
+    return a * b;
```
Comment (Dilstom) [inline assignments/objects.js:60]: hmm, did it work for you?
Do you need space between `function (a,b)`?

**Review by Dilstom — CHANGES_REQUESTED** [review]

File: assignments/arrays.js (hunk 4 above)
Comment (paulabernathy7) [reply assignments/arrays.js:107]: It worked for me

File: assignments/objects.js (hunk 7 above)
Comment (paulabernathy7) [reply assignments/objects.js:60]: Yes it worked

Comment (Dilstom) [reply assignments/objects.js:60]: you are right! my extension removes this space automatically.

Comment (Dilstom) [reply assignments/objects.js:60]: http://crockford.com/javascript/code.html#function

File: assignments/arrays.js (hunk 4 above)
Comment (Dilstom) [reply assignments/arrays.js:107]: what result you are getting )

Comment (paulabernathy7) [reply assignments/arrays.js:107]: ah, you're right it doesn't work how I want.

File: assignments/arrays.js [hunk 8]
```diff
@@ -103,7 +103,7 @@ console.log(oldCars.length);
 let BMWAndAudi = [];
 
 for (let i = 0; i < inventory.length; i ++) {
-    if (inventory[i].car_make === "BMW" || "Audi"){
+    if (inventory[i].car_make === "BMW" || inventory[i].car_make === "Audi"){
```
Comment (Dilstom) [inline assignments/arrays.js:106]: nice!

**Review by Dilstom — APPROVED** [review]

**Pull request merged**